### ISSUE TITLE
e2e tests should check that extension is installed and activated

### ISSUE THREAD

**Issue by dgolovin** [redhat-developer/podman-desktop-redhat-account-ext#292]
Migration to pnpm lead to the problems during the build that weren't noticed right away. See https://github.com/redhat-developer/podman-desktop-redhat-account-ext/pull/290. If I install nightly build from github activation fails with error.

```
Error: Cannot find module 'object-hash' Require stack: - /Users/esk/.local/share/containers/podman-desktop/plugins/ghcrioredhatdeveloperpodmandesktopredhataccountext/dist/extension.cjs - /Users/rsk/Sources/podman-desktop/packages/main/dist/index.cjs
Stack trace:
Error: Cannot find module 'object-hash'
Require stack:
- /Users/esk/.local/share/containers/podman-desktop/plugins/ghcrioredhatdeveloperpodmandesktopredhataccountext/dist/extension.cjs
- /Users/esk/Sources/podman-desktop/packages/main/dist/index.cjs
    at Module._resolveFilename (node:internal/modules/cjs/loader:1152:15)
    at s._resolveFilename (node:electron/js2c/browser_init:2:123445)
    at Module._load (node:internal/modules/cjs/loader:993:27)
    at c._load (node:electron/js2c/node_init:2:17025)
    at Function.load2 [as _load] (/Users/esk/Sources/podman-desktop/packages/main/dist/index.cjs:78751:29)
    at Module.require (node:internal/modules/cjs/loader:1240:19)
    at require (node:internal/modules/helpers:179:18)
    at Module.<anonymous> (/Users/esk/.local/share/containers/podman-desktop/plugins/ghcrioredhatdeveloperpodmandesktopredhataccountext/dist/extension.cjs:18:22)
    at Module._compile (node:internal/modules/cjs/loader:1373:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1432:10)
```

This error occurs because node_modules under dist folder for external dependencies is not created during the build and pr above taking care of it.

e2e test at the same time works fine, even with extension is not activated due to error.

**Comment by odockal** [comment]
@dgolovin I think that e2e tests are not omitting something. It is that we run tests locally (on ci agent) from built sources and so the problem is not there. In order to make extension to fail to load we would need to rewrite the e2e tests to run against built production binary.

**Comment by odockal** [comment]
Fixed by #300
